Integrations/FireEyeHelix/FireEyeHelix.py

```diff
@@ -0,0 +1,1643 @@
+            created_at: Creation date of the list.
+            description: Description of the list.
+            is_active: Set to true if the list is active.
+            is_internal: Set to true if the list is internal.
+            is_protected: Set to true if list is protected.
+            name: Name of the list.
+            short_name: Short name of the list.
+            type: Type of the list.
+            updated_at: The time the list was last updated at.
+            usage: Multiple values may be separated by commas.
+            order_by: Which field to use when ordering the results.
+
+        Returns:
+            Response from API.
+        """
+        suffix = '/api/v3/lists'
+        params = assign_params(
+            limit=limit,
+            offset=offset,
+            created_at=created_at,
+            description=description,
+            is_active=is_active and is_active != 'false',
+            is_internal=is_internal and is_internal != 'false',
+            is_protected=is_protected and is_protected != 'false',
+            name=name,
+            short_name=short_name,
+            type=type,
+            updated_at=updated_at,
+            usage=usage,
+            order_by=order_by
+        )
+        return self._http_request('GET', suffix, params=params)
+
+    def get_list_by_id(self, list_id: Optional[Any]) -> Dict:
+        """Get a list by id via a GET request
+
+        Args:
+            list_id: ID of the list
+
+        Returns:
+            Response from API.
+        """
+        suffix = f'/api/v3/lists/{list_id}'
+        return self._http_request('GET', suffix)
+
+    def create_list(self, name: str, usage: str = None, short_name: str = None, is_internal: Union[str, bool] = None,
+                    is_active: Union[str, bool] = None, is_protected: Union[str, bool] = None,
+                    is_hidden: Union[str, bool] = None, type: str = None, description: str = None, **kwargs) -> Dict:
+        """Creates a list using a POST request
+
+        Args:
+            name: Name of the list.
+            short_name: Short name of the list.
+            is_internal: Boolean flag for is internal.
+            is_active: Boolean flag for is active.
+            is_protected: Boolean flag for is protected.
+            is_hidden: Boolean flag for is hiddden.
+            usage: Usage of the list.
+            type: Type of the list.
+            description: Description of the list.
+
+        Returns:
+            Response from API.
+        """
+        suffix = '/api/v3/lists'
+        body = assign_params(
+            name=name,
+            short_name=short_name,
+            is_internal=is_internal != 'false' if is_internal else is_internal,
+            is_active=is_active != 'false' if is_active else is_active,
+            is_protected=is_protected != 'false' if is_protected else is_protected,
+            is_hidden=is_hidden != 'false' if is_hidden else is_hidden,
```

Comment:
If we will remove the unions we could remove this conditions here.

this is a recurring pattern that should be changed.

Reply:
👍